resources/backtick/migrations/106-cron.up.sql

```diff
@@ -0,0 +1,55 @@
+ALTER TABLE backtick_recurring ADD COLUMN cronspec text;
+
+ALTER TABLE backtick_recurring ALTER COLUMN interval DROP NOT NULL;
+
+ALTER TABLE backtick_recurring
+ADD CHECK ((interval IS NULL OR cronspec IS NULL) AND
+           (interval IS NOT NULL OR cronspec IS NOT NULL));
+
+-- See migration 104-recurring.up.sql
+CREATE OR REPLACE FUNCTION
+  backtick_upsert_interval(the_name TEXT,
+                           the_interval integer,
+                           the_cronspec text,
+                           the_next timestamp)
+  RETURNS VOID AS
+$$
+  BEGIN
+    LOOP
+      -- A existing job, with the same interval/cronspec => Do nothing
+      IF EXISTS (SELECT 1 FROM backtick_recurring
+                 WHERE name = the_name
+                 AND interval IS NOT DISTINCT FROM the_interval
```

Comment:
You mean the `IS NOT DISTINCT FROM` part?

Basically it's equivalence but works with `NULL` values as well. See: https://www.postgresql.org/docs/current/static/functions-comparison.html